### DISCUSSION TITLE
Missing code?

### DISCUSSION THREAD

**Discussion by saif-o99** [edumeet/edumeet-client#157] [Q&A]
Hey guys, i'm looking at the demo trying to look at the spotlight logic.
but i can't find the full code of the spotlight in this repo, is there any reason? 
is the demo on another repo?
for example i dont find a usage for these reducers:
![image](https://github.com/edumeet/edumeet-client/assets/52207667/853e5406-6174-4917-8a54-581e54e3daf1)

**Comment by havfo** [comment]
The logic to put a peer into spotlight is not fully developed yet. We will expand the media layout functionality further in the future.
